### PR TITLE
Adjust headings and spacings to align with magnetic

### PR DESCRIPTION
**Pull request by rwharrisjr**
#499

### CODE REVIEW

File: framework/styles/utilities/variables.scss [hunk 1]
```diff
@@ -7,20 +7,20 @@ $font-family-monospace: SFMono-Regular, Menlo, Monaco, Consolas,
...
-$font-size--lg: rem(1.25) !default; // 20px
-$font-size--md: rem(1.125) !default; // 18px
-$font-size--sm: rem(0.875) !default; // 14px
-$font-size--xs: rem(0.75) !default; // 12px
+$font-size--xl: rem(1.125) !default; // 18px
+$font-size--lg: rem(1) !default; // 16px
+$font-size--md: rem(0.875) !default; // 14px
+$font-size--sm: rem(0.75) !default; // 12px
+$font-size--xs: rem(0.625) !default; // 12px
 $font-size--xxs: rem(0.625) !default; // 10px
```
Comment (androche) [inline framework/styles/utilities/variables.scss:15]: Can delete

File: framework/styles/utilities/variables.scss [hunk 2]
```diff
@@ -7,20 +7,20 @@ $font-family-monospace: SFMono-Regular, Menlo, Monaco, Consolas,
...
-$font-size--xl: rem(1.5) !default; // 24px
-$font-size--lg: rem(1.25) !default; // 20px
-$font-size--md: rem(1.125) !default; // 18px
-$font-size--sm: rem(0.875) !default; // 14px
-$font-size--xs: rem(0.75) !default; // 12px
+$font-size--xl: rem(1.125) !default; // 18px
+$font-size--lg: rem(1) !default; // 16px
+$font-size--md: rem(0.875) !default; // 14px
+$font-size--sm: rem(0.75) !default; // 12px
+$font-size--xs: rem(0.625) !default; // 12px
```
Comment (androche) [inline framework/styles/utilities/variables.scss:14]: Change comment to 10px

File: framework/styles/utilities/variables.scss [hunk 3]
```diff
@@ -7,20 +7,20 @@ $font-family-monospace: SFMono-Regular, Menlo, Monaco, Consolas,
...
 // Font-size: h1-h4
 // -----------------------------------------------------------------------------
 $h1-font-size: rem(1.5); // 24px
 $h2-font-size: rem(1.125) !default; // 20px
-$h3-font-size: rem(1.1) !default; // 18px
-$h4-font-size: rem(0.875) !default; // 16px
-$h5-font-size: rem(0.625) !default; // 14px
+$h3-font-size: rem(1) !default; // 16px
+$h4-font-size: rem(0.875) !default; // 14px
+$h5-font-size: rem(0.625) !default; // 12px
```
Comment (androche) [inline framework/styles/utilities/variables.scss:23]: rem(0.75) for 12px

File: framework/styles/utilities/variables.scss [hunk 4]
```diff
@@ -30,9 +30,13 @@ $font-weight-bold: 700 !default;
...
-$font-weight--bold: bold !default;
-$line-height: 1.5 !default;
-$line-height--lg: rem(1.5) !default; // 24px
+$font-weight--bold: $font-weight-bold !default;
+$line-height: 1.25 !default;
+$line-height--xl: rem(1.5) !default; // 24px
+$line-height--lg: rem(1.375) !default; // 22px
+$line-height--md: rem(1.25) !default; // 20px
+$line-height--sm: rem(1.125) !default; // 18px
+$line-height--xs: rem(1) !default; // 24px
```
Comment (androche) [inline framework/styles/utilities/variables.scss:39]: This is 16px

File: framework/styles/utilities/variables.scss [hunk 5]
```diff
@@ -59,9 +63,15 @@ $spacers: () !default;
...
-$headings-line-height: 1.4 !default;
-$headings-margin: $spacer * 2 !default;
+$headings-margin: $spacer * 3 !default;
 $headings-color: inherit !default;
+
+$h1-line-height: rem(2.125); // 34px
+$h2-line-height: rem(1.5) !default; // 24px
+$h3-line-height: rem(1.375) !default; // 22px
+$h4-line-height: rem(1.25) !default; // 20px
+$h5-line-height: rem(0.625) !default; // 14px
```
Comment (androche) [inline framework/styles/utilities/variables.scss:73]: This converts to 10px

File: framework/styles/utilities/variables.scss (hunk 1 above)
Comment (rwharrisjr) [reply framework/styles/utilities/variables.scss:15]: for good measure I think we should leave these in, incase anything else needs to reference them externally

**Review by androche — APPROVED** [review]
It looks pretty good to me from clicking around. The only thing that stood out was something about the a-row is giving a little more padding in an area. It could just be not the best use of a-row in this case though.

Screenshot 2023-08-28 at 3 24 19 PM

**Comment by rwharrisjr** [comment]
> It looks pretty good to me from clicking around. The only thing that stood out was something about the a-row is giving a little more padding in an area. It could just be not the best use of a-row in this case though.
> 
> Screenshot 2023-08-28 at 3 24 19 PM

The page header spacing should be standardized by future components

**Pull request merged**